lib/src/base-styles/_nx-lists.scss

```diff
@@ -126,6 +126,10 @@
  */
 .nx-list--clickable {
   .nx-list__item {
+    border-bottom: 1px solid transparent;
+    border-left: 1px solid transparent;
+    border-right: 1px solid transparent;
```

Comment:
Should we remove a pixel from the padding to counteract the extra pixel in this border? That's what I'm doing in nx-table.

Reply:
> Should we remove a pixel from the padding to counteract the extra pixel in this border? That's what I'm doing in nx-table.

That's why I didn't really like the idea of adding the borders to all of the lists, I don't care if the clickable lists are 2px taller, they're already different, but now we've made them all 2px taller

Reply:
> Should we remove a pixel from the padding to counteract the extra pixel in this border? That's what I'm doing in nx-table.

Adjusted here: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/c82687e37fd032042945b36c7515bc013c7a6f33